constellations/gemini-admin/src/help/help.sass

```diff
@@ -1,24 +1,23 @@
 @import "../_.constants.sass"
 
-body
-  margin: 0
-  h1
-    margin-left: 100px
-    margin-top: 30px
-    font-family: $fontFamilyRoboto
-    font-size: $fontSize4
-    color: $colorTurquoise
+#view-help
+    h1
+        color: $colorTurquoise
+        margin-left: 100px
+        margin-top: 30px
+        font-family: $fontFamilyRoboto
+        font-size: $fontSize4
     
-  p
-    margin: 20px 0px 40px 100px
-    width: 1000px
-    font-family: $fontFamilyRoboto
-    color: $colorGrayDark
+    p
+        margin: 20px 0px 40px 100px
+        width: 1000px
+        font-family: $fontFamilyRoboto
+        color: $colorGrayDark
     
-  h2
-    font-family: $fontFamilyRoboto
-    font-size: $fontSize4
-    margin-left: 100px
-    padding-bottom: 20px
-    border-bottom: 1px solid
-    width: 1000px
+    h2
+        font-family: $fontFamilyRoboto
+        font-size: $fontSize4
+        margin-left: 100px
+        padding-bottom: 20px
+        border-bottom: 1px solid
+        width: 1000px
```

Comment:
There are many predefined lengths in `_constants.sass` already, so you should probably use those. For example, `$spacing128x` is 1024 px.